content/security/docs/iam.md

```diff
@@ -301,47 +301,7 @@ resource "aws_launch_template" "foo" {
 
 You can also block a pod's access to EC2 metadata by manipulating iptables on the node. For further information about this method, see [Limiting access to the instance metadata service](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html#instance-metadata-limiting-access).
 
-If you have an application that is using an older version of the AWS SDK that doesn't support IRSA, and you want the pod to inherit the role assigned to the instance, consider using Kubernetes network policies to **selectively** allow access EC2 metadata.
-
-Start with a policy that blocks access to the metadata service from all Pods:
-
-```yaml
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: deny-metadata-access
-  namespace: example
-spec:
-  podSelector: {}
-  policyTypes:
-  - Egress
-  egress:
-  - to:
-    - ipBlock:
-        cidr: 0.0.0.0/0
-        except:
-        - 169.254.169.254/32
-```
-
-Then allow access from select pods by adding following policy, modifying the `PodSelector` as appropriate.
-
-```yaml
-apiVersion: networking.k8s.io/v1
-kind: NetworkPolicy
-metadata:
-  name: allow-metadata-access
-  namespace: example
-spec:
-  podSelector:
-    matchLabels:
-      app: myapp  
-  policyTypes:
-  - Egress
-  egress:
-  - to:
-    - ipBlock:
-        cidr: 169.254.169.254/32
-```
+If you have an application that is using an older version of the AWS SDK that doesn't support IRSA, you should update the SDK version.
```

Comment:
I think this is a good recommendation, but it may not always be feasible to upgrade the SDK version. Is there another way we can phrase this?

Reply:
Kind of depends. While it may not always be feasible, I wonder how few or how many those cases are. If it's a very small minority, might make sense to keep it as is. If it's a notable minority, what would our recommendation be instead?